mapdata/WarcraftLegacies/UpgradeData/Skin/1651402066.json

```
@@ -38,7 +38,7 @@
       "Level": 1,
       "Id": 828536167,
       "Type": 3,
-      "Value": "Upgrades all Ancients\u0027 movement speed by 40, health points and armor. Reduces Summon Treants cooldown by 20 seconds."
+      "Value": "Upgrades all Ancients\u0027 movement speed by 20, health points and armor. Reduces Summon Treants cooldown by 10 seconds."
```

Let's just fix this jank grammar while we're at it...

```suggestion
      "Value": "Improves all Ancients by increasing their movement speed by 20, hit points by 10%, and armor by a small amount. Reduces Summon Treants\u0027 cooldown by 10 seconds."
```